cluster/charts/fuel-local/Chart.yaml

```diff
@@ -1,5 +1,12 @@
-apiVersion: v1
+apiVersion: v2
 appVersion: "1.0"
 description: A Helm chart for Kubernetes
 name: fuel-local
 version: 0.2.0
+dependencies:
+  - name: fuel-streams-publisher
```

Comment:
By using publisher and nats charts here, we avoid having duplicated stuff and we can leverage the same charts we will use to deploy to production